src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -181,16 +254,9 @@ impl WorkUnitInnerStore {
 impl WorkunitStore {
   pub fn new(rendering_dynamic_ui: bool) -> WorkunitStore {
     WorkunitStore {
+      streaming_workunit_data: StreamingWorkunitData::new(),
+      heavy_hitters_data: HeavyHittersData::new(),
```

Comment:
If you're not rendering the UI, or not recording streaming_workunit_data, these could each be `None`. But also fine to do in a followup.

Reply:
Yeah, I'll respond to these points in a followup commit, I think these are generally good ideas.